src/app/navbar/navbar.component.ts

```diff
@@ -67,5 +68,16 @@ export class NavbarComponent implements OnInit {
     toggleInfokit() {
         this.infoPop.emit();
     }
+
+    editUser(): void {
+      const dialogConfig = new MatDialogConfig();
+      dialogConfig.data = {
+        profilePicture: this.proPic,
+        username: this.username
+      };
+      dialogConfig.width = '600px';
```

Comment:
is this responsive?

Reply:
Surprisingly, it turned out to be responsive!! I think this width property is = max-width here.